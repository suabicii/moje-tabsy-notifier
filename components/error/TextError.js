import React from "react";
import {Text, View} from "react-native";

function TextError({content}) {
    return (
        <View>
            <Text style={{
                color: "#ff5252",
                fontSize: 20,
                fontWeight: "200",
                marginBottom: 20,
                textAlign: "center"
            }}>
                {content}
            </Text>
        </View>
    );
}

export default TextError;