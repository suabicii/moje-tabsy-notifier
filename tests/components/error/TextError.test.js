import React from "react";
import renderer from "react-test-renderer";
import TextError from "../../../components/error/TextError";

it('should correctly render TextError component with some content', () => {
    const tree = renderer.create(<TextError content="Dummy error"/>).toJSON();
    expect(tree).toMatchSnapshot();
});