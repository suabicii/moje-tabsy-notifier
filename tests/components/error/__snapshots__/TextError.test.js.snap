// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render TextError component with some content 1`] = `
<View>
  <Text
    style={
      {
        "color": "#ff5252",
        "fontSize": 20,
        "fontWeight": "200",
        "marginBottom": 20,
        "textAlign": "center",
      }
    }
    testID="error"
  >
    Dummy error
  </Text>
</View>
`;
